tests/rundoctest.py

```diff
@@ -8,12 +8,18 @@
 path = os.path.dirname(__file__)
 if path:
     os.chdir(path)
+os.chdir('..')
```

Comment:
If I understand correctly what `pardir` does, it would rather be

```
path = os.path.abspath(os.path.join(os.path.dirname(__file__), os.path.pardir))
os.chdir(path)
sys.path.append(path)
```
longer, but maybe more robust?